src/network/clientpackethandler.cpp

```diff
@@ -410,10 +410,10 @@ void Client::handleCommand_Movement(NetworkPacket* pkt)
 	Player *player = m_env.getLocalPlayer();
 	assert(player != NULL);
 
-	float mad, maa, maf, msw, mscr, msf, mscl, msj, lf, lfs, ls, g;
+	float mad, maa, maf, msw, mscr, msf, mscl, msj, lf, lfs, ls, g, dmg;
 
 	*pkt >> mad >> maa >> maf >> msw >> mscr >> msf >> mscl >> msj
-		>> lf >> lfs >> ls >> g;
+		>> lf >> lfs >> ls >> g >> dmg;
```

Comment:
If you change the protocol, you have to make it in a way that nothing breaks. You should do:

1. let client and server check whether the other's protocol counter is greater or equal than 25.
2. document your change in networkprotocol.h, both at the start of the file, where the protocol version 25 is explained, and where the packet itself is defined. I'm not sure if @nerzhul requires to add "// Added in protocol version 25" notes.